samples/uploadDirectory.js

```diff
@@ -36,64 +36,44 @@ async function main(bucketName, directoryPath) {
   // Creates a client
   const storage = new Storage();
 
-  const fs = require('fs');
+  const fs = require('fs/promises');
```

Comment:
It looks like before node 14 you need to do `require('fs').promises`

Reply:
Thanks, I ran into another issue w/ `require('fs').promises` where `eslint` believes its not available in 10 haha I just used `util.promisify` for now until